src/map/status.c

```diff
@@ -2572,12 +2572,27 @@ static int status_calc_pc_(struct map_session_data *sd, enum e_status_calc_opt o
 
 	status->calc_pc_additional(sd, opt);
 
-	if( sd->pd ) { // Pet Bonus
+	if (sd->pd != NULL) { /// Pet Bonuses.
 		struct pet_data *pd = sd->pd;
-		if( pd && pd->petDB && pd->petDB->equip_script && pd->pet.intimate >= battle_config.pet_equip_min_friendly )
-			script->run(pd->petDB->equip_script,0,sd->bl.id,0);
-		if( pd && pd->pet.intimate > 0 && (!battle_config.pet_equip_required || pd->pet.equip > 0) && pd->state.skillbonus == 1 && pd->bonus )
-			pc->bonus(sd,pd->bonus->type, pd->bonus->val);
+
+#ifndef RENEWAL
+		if (pd != NULL && pd->petDB != NULL && pd->petDB->equip_script != NULL &&
+		    pd->pet.intimate >= battle_config.pet_equip_min_friendly)
+			script->run(pd->petDB->equip_script, 0, sd->bl.id, 0);
+#else
+		/**
+		 * [Kenpachi]
+		 * The intimacy will be removed, when pet DB gets updated, since the intimacy level verification will be done in the pet's 'EquipScript' field then.
+		 *
```

Comment:
Why it will be removed?

Reply:
That's a type. Not the intimacy will be removed, but the check for its level (`pd->pet.intimate >= PET_INTIMACY_CORDIAL`).